tests/validation/response/test_response_status_code_validation.py

```diff
@@ -43,3 +43,31 @@ def test_response_parameter_validation():
         err.value.messages[0]['status_code'][0],
         MESSAGES['response']['invalid_status_code'],
     )
+
+def test_response_paramater_uses_default():
+    """
+    Test that a `default` key is used if one exists and no matching response is found.
+
+    See http://swagger.io/specification/#responsesObject.
+    """
+
+    schema = SchemaFactory(
+        paths={
+            '/get': {
+                'get': {
+                    'responses': {
+                        200: {'description': 'Success'},
+                        'default': {'description': 'Unexpected error.'}
+                    },
+                },
+            },
+        },
+    )
+
+    response = ResponseFactory(url='http://www.example.com/get', status_code=301)
+
+    validate_response(
```

Comment:
If you are referring to assertion vs exception, I do not.  As long as the test suite will not pass if this test is failing.